test/create-article.ts

```diff
@@ -8,7 +8,7 @@ import { rdf, schema } from '../src/namespaces';
 
 type Options = {
   id?: QuadSubject;
-  name?: Literal;
+  name?: Literal | null;
```

Comment:
only example that needs `null` in addition to the `undefined` provided by `?`?

Reply:
https://github.com/libero/article-store/blob/a6762f995cdd3c277786b858e789a17282b2790a/test/routes/add-article.test.ts#L55-L56

Reply:
So it's by choice, but it's an error case so wouldn't expect it to mean something in particular like "a title that is currently unknown"